### DISCUSSION TITLE
How to export/import settings in order to switch to msstore version

### DISCUSSION THREAD

**Discussion by HolgerAusB** [mifi/lossless-cut#2167] [Support]
High there,

after using the free version for a while, I am now planning to buy the msstore version. My question is, how I could do this without losing my settings?

I installed/unziped LosslessCut in a non-standard path at the moment.

Will the msstore version find the settings of free version automatically (because they are stored in the registry?) or should I locate the settings somewhere in the file system and make a backup?

**Comment by mifi** [comment]
Hi. MS Store apps save app settings in a separate location, so it won't automatically copy settings over. However you should be able to manually copy over the `config.json` file.

- On the github version of lsoslesscut, `config.json` should be stored in `%APPDATA%\LosslessCut\config.json`.
- On the MS Store version, I'm not 100% sure but I think `config.json` is stored in: `C:\Users\%USERNAME%\AppData\Local\Packages\57275mifi.no.LosslessCut_eg8x93dt4dxje\LocalCache\Roaming\LosslessCut\config.json`

So here's what I would do:

1. First start the MS Store version of LossslessCut
2. Find the exact location of its `config.json` file (see above)
3. Exit LosslessCut
4. Find the config.json file from the github version
5. Copy the github version config.json file to the MS Store folder and replace the existing file
6. Start the MS Store version and check that settings got copied correctly.

More info: https://github.com/mifi/lossless-cut/blob/master/installation.md#settings-and-temporary-files

I will update this page with this info:

https://github.com/mifi/lossless-cut/blob/master/installation.md#settings-and-temporary-files

I will also make a new menu item in the Help menu that can be used to locate the config file to make this easier in the future.

Hope this helps!

**Reply by HolgerAusB** [reply]
in fact there is no 'LosslessCut' folder under `...\57275mifi.no.LosslessCut_eg8x93dt4dxje\LocalCache\Roaming\` for the MS Store app after first start.

But luckily the just bought store app found my existing config from the github version. So it might be the same location.

**Reply by mifi** [reply]
Oh, so you're saying that the MS Store version shares config path with the github version? Then I must have been totally wrong. But that's good to known, thanks!

**Reply by HolgerAusB** [reply]
In my case: yes, it seem so.

But it could be different, if one is installing LosslessCut from msstore without having previously installed a github version. Maybe the app is looking-up on several places for an existing config or it is having a registry entry to point on the config. Though I didn't find a hint to last guess with regedit.

**Reply by mifi** [reply]
Config is not stored in the registry - it's a simple flat file. There is no logic in the app for checking for existing an config file, so it must be that it's using the same file/path.
